02.calendar/cal.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+require "date"
+require 'optparse'
+
+opt = OptionParser.new
+options = {}
+opt.on('-m [month]', Integer) {|v| options[:m] = v}
+opt.on('-y [year]', Integer) {|v| options[:y] = v}
+opt.parse(ARGV)
+
+month = options[:m]
+year = options[:y]
+
+today = Date.today
+if month == nil
+  month = today.month
+end
```

Comment:
```suggestion
month ||= today.month
```

でも同じ動きになります。

もっというと、

```ruby
month = options[:m] || today.month
```

でもOKです。（ちょっと高等テクニックですが）

Reply:
私がこれができたらなぁと思っていたのがまさにこの書き方です！　教えて頂きありがとうございます！
[【Ruby】使いこなせると便利。||演算子のいろんな使い方](https://qiita.com/ngron/items/4d3ca9827d1969fccd60)
少し古い記事ですが、こちらを読んで理解を深めてからコードに落とし込みました。